servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -1570,6 +1578,18 @@ public void setState(long fileId, SetStateOptions options) throws FileDoesNotExi
    * @throws FileDoesNotExistException when the file does not exist
    */
   public long scheduleAsyncPersistence(long fileId) throws FileDoesNotExistException {
+    long workerId = scheduleAsyncPersistenceInternal(fileId);
+
+    // write to journal
+    AsyncPersistRequestEntry asyncPersistRequestEntry =
+        AsyncPersistRequestEntry.newBuilder().setFileId(fileId).build();
+    writeJournalEntry(
```

Comment:
I think all journal writes and journal flushes are synchronized on mInodeTree. I'm not sure what will happen if this one is not protected by the same lock.